docs/src/functions.md

```diff
@@ -1,4 +1,4 @@
-# Types and functions
+# API reference
 
```

Comment:
A detailed listing of the functions can be seen here (link)

Reply:
Done below!

Reply:
yep that's in another file